Services/Dashboard/Administration/classes/class.ilObjDashboardSettingsGUI.php

```diff
@@ -238,63 +227,56 @@ public function setSettingsSubTabs(string $a_active): void
     protected function editViewCoursesGroups(): void
     {
         if ($this->settings->get('mmbr_my_crs_grp', '0') !== '1') {
-            $content[] = $this->ui->factory()->messageBox()->info($this->lng->txt('memberships_disabled_info'));
+            $content[] = $this->ui_factory->messageBox()->info($this->lng->txt('memberships_disabled_info'));
         }
-        $this->tabs_gui->activateTab("settings");
-        $this->setSettingsSubTabs("view_courses_groups");
+        $this->tabs_gui->activateTab('settings');
+        $this->setSettingsSubTabs('view_courses_groups');
 
         $content[] = $this->getViewSettingsForm($this->viewSettings->getMembershipsView());
         $this->tpl->setContent($this->ui_renderer->render($content));
     }
 
-    protected function getViewSettingsForm(int $view): \ILIAS\UI\Component\Input\Container\Form\Standard
+    protected function getViewSettingsForm(int $view): Standard
     {
-        $ctrl = $this->ctrl;
         $lng = $this->lng;
         $ui_factory = $this->ui_factory;
 
-        if ($view == $this->viewSettings->getSelectedItemsView()) {
-            $save_cmd = "saveViewFavourites";
+        if ($view === $this->viewSettings->getSelectedItemsView()) {
+            $save_cmd = 'saveViewFavourites';
         } else {
-            $save_cmd = "saveViewCoursesGroups";
+            $save_cmd = 'saveViewCoursesGroups';
         }
 
-        // presentation
         $ops = $this->viewSettings->getAvailablePresentationsByView($view);
-        $pres_options = array_column(array_map(function ($k, $v) use ($lng) {
-            return [$v, $lng->txt("dash_" . $v)];
-        }, array_keys($ops), $ops), 1, 0);
-        $avail_pres = $ui_factory->input()->field()->multiSelect($lng->txt("dash_avail_presentation"), $pres_options)
+        $pres_options = array_column(array_map(static fn ($k, $v) => [$v, $lng->txt('dash_' . $v)], array_keys($ops), $ops), 1, 0);
```

Comment:
Please remove the unused extra array:
```suggestion
        $pres_options = array_column(array_map(static fn ($v) => [$v, $lng->txt('dash_' . $v)], $ops), 1, 0);
```